src/app/static/src/js/components/runWorkflow/runWorkflowParameters.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+    <div v-if="hasParams(report)">
+        <p v-for="(param, paramIndex) in report.param_list"
+           :key="paramIndex"
+           class="non-default-param">{{ param.name }}: {{ param.value }}</p>
+        <p v-if="!report.param_list.length">
+            No non-default parameters
+        </p>
+        <div v-if="report.default_param_list.length > 0"
+             :id="`default-params-${index}`">
+            <b-link v-b-toggle="`collapseSummary-${index}`"
+                    href="#"
+                    class="show-defaults pt-2 d-inline-block small">
+                <span class="when-closed">Show</span>
+                <span class="when-open">Hide</span> defaults...
+            </b-link>
+            <b-collapse :id="`collapseSummary-${index}`">
+                <p v-for="(param, paramIndex) in report.default_param_list"
+                   :id="`default-params-collapse-${index}-${paramIndex}`"
+                   :key="paramIndex">{{ param.name }}: {{ param.value }}</p>
+            </b-collapse>
+        </div>
+    </div>
+    <p v-else>
+        No parameters
+    </p>
+</template>
+
+<script lang="ts">
+    import Vue from 'vue'
+    import {WorkflowRunReportStatus, WorkflowReportWithDependencies} from "../../utils/types";
+    import {BLink} from "bootstrap-vue/esm/components/link";
+    import {BCollapse} from "bootstrap-vue/esm/components/collapse";
+    import {VBToggle} from 'bootstrap-vue';
+    import {hasParams} from "../../utils/helpers.ts";
+
+    interface Props {
+        report: WorkflowRunReportStatus;
```

Comment:
I don't think this is quite the right type? Both runWorkflowTable and workflowSummaryReports pass this component a report from workflowSummary, which is actually a WorkflowReportWithDependencies. In fact you're using that type in the `hasParams` method - presumably that could just be a computed, once the prop's type is corrected?